doc/source/serve/production.md

```diff
@@ -583,3 +565,218 @@ This section summarizes the best practices when deploying to production:
 * Use `serve config` to check the latest config that your Serve application received. This is its goal state.
 * Make lightweight configuration updates (e.g. `num_replicas` or `user_config` changes) by modifying your Serve config file and redeploying it with `serve deploy`.
 * Make heavyweight code updates (e.g. `runtime_env` changes) by starting a new Ray cluster, updating your Serve config file, and deploying the file with `serve deploy` to the new cluster. Once the new deployment is finished, switch your traffic to the new cluster.
+
+(serve-in-production-kubernetes)=
+
+## Deploying to Production on Kubernetes
+
+The above sections walk through how to deploy a Serve application manually to a running Ray cluster using the Serve CLI.
+Ray Serve can also be run on Kubernetes, providing the best of both worlds: the user experience and scalable compute of Ray Serve and the production-readiness of Kubernetes.
```

Comment:
This makes it sound like Ray Serve is not production-ready :D

Reply:
hmm yeah I can see that. let me word it a bit more carefully...